packages/mobile/src/app/reducers.ts

```diff
@@ -23,6 +23,9 @@ export interface State {
   hideVerification: boolean
   showRaiseDailyLimitTarget: string | undefined
   walletConnectEnabled: boolean
+  // In 1.13 we had a critical error which requires a migration to fix. See |verificationMigration.ts|
+  // for the migration code. We can remove all the code associated with this after some time has passed.
+  ranVerificationMigration: boolean
```

Comment:
What do you think of using a timestamp instead?
https://changelog.com/posts/you-might-as-well-timestamp-it
```suggestion
  ranVerificationMigrationAt: number | undefined 
```